examples/js/web-worker/README.md

```diff
@@ -0,0 +1,37 @@
+# Web-Worker
+
+This is an example project showing how to use `@bugsnag/web-worker` with a web worker or service worker.
+
+The project contains a single page which spawns 2 web workers and 1 service worker. Each worker has its own instance of `@bugsnag/web-worker`, and the web page itself is using an instance of `@bugsnag/js`. 
+
+By default the BugSnag `web-worker` package will not report unhandled errors from the worker script. If you have BugSnag running in the worker’s host web app this will leave unhandled errors to propagate up to the browser context. As these errors are in the same context as the BugSnag sessions, they will impact the stability score. However they will not contain any breadcrumbs or metadata added from within the worker context – these will only be present in handled errors.
+
+If you choose to enable automatic error detection in the `web-worker` using the [`autoDetectErrors` configuration option](https://docs.bugsnag.com/platforms/javascript/configuration-options/#autodetecterrors), the errors reported from within the worker will have the breadcrumbs and metadata set within the worker, but these errors will not be tied to the browser sessions and will not impact the stability score.
+
+Similarly, the BugSnag `web-worker` package does not report sessions by default. If you are using a worker hosted in your own web app the sessions should be managed by the host web app as it is this that users engage with directly. However, if you wish to treat the worker independently from the host web app, automatic session tracking can be enabled in the `web-worker` package using the [`autoTrackSessions` configuration option](https://docs.bugsnag.com/platforms/javascript/configuration-options/#autotracksessions).
```

Comment:
This is a more complicated picture than we first thought – to avoid out-of-date and complicated content here, I suggest we just link to the docs:

```suggestion
BugSnag can be configured to run in different ways inside a worker depending on whether it's a web or service worker and also its relationship with the “host” web app that spawns it. For full details on this, please see our [online docs](https://docs.bugsnag.com/platforms/javascript/web-workers/#reporting-unhandled-errors).

The project contains a single page which spawns 2 web workers and 1 service worker. Each worker has its own instance of `@bugsnag/web-worker`, and the web page itself is using an instance of `@bugsnag/js`.
```